packages/react-refresh/src/__tests__/ReactFresh-test.js

```diff
@@ -1411,19 +1416,19 @@ describe('ReactFresh', () => {
         $RefreshReg$(Hello, 'Hello');
       });
 
-      // Colors inside both trees should change:
+      // Only update color in the visible child
       expect(container.childNodes.length).toBe(2);
       expect(container.childNodes[0]).toBe(primaryChild);
       expect(container.childNodes[1]).toBe(fallbackChild);
       expect(primaryChild.textContent).toBe('Content 1');
-      expect(primaryChild.style.color).toBe('red');
+      expect(primaryChild.style.color).toBe('green');
```

Comment:
This test is no longer passing. Seems like it really can't patch both trees anymore. @gaearon Could you take a look if there's a legit bug or something we just can't test in new roots anymore.

Reply:
I'm not sure but I suspect maybe we don't bother re-rendering into the invisible component because it's offscreen? I'd rephrase the test so that it doesn't assert that it *has to* be green or red here — I think the important part is that by the time we unsuspend, the color of the content is up-to-date. Until then, I think it doesn't really matter because it's unobservable.

Reply:
Test now reads "only patches the fallback tree while suspended"